source/documentation/index.md

```diff
@@ -4,4 +4,118 @@ GOV.UK Platform as a Service (PaaS) is a cloud-hosting platform being built by t
 
 GOV.UK PaaS is currently in private beta.
 
-If you’d like to be one of the early applications going live on the platform, please contact us by emailing [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk). There is limited availability for these early applications.
\ No newline at end of file
+GOV.UK PaaS is built using the open source [Cloud Foundry](https://www.cloudfoundry.org/) project, and runs on Amazon Web Services. [Read our blog post](https://governmentasaplatform.blog.gov.uk/2015/12/17/choosing-cloudfoundry/) for more information on why we chose Cloud Foundry. Refer to the [features page](https://www.cloud.service.gov.uk/features) for more information on the GOV.UK PaaS.
+
+If you want to use the platform to host your app, contact us by emailing [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk).
+
+# Before you start
+
+To start using the platform, your department, agency or team must agree to the Terms of Use.
+
+To be hosted on the GOV.UK PaaS, your app must:
+
+- follow the [twelve-factor application principles](/#12-factor-application-principles); this will be true if your app was written to be deployed to another PaaS like Heroku
+- [store data classified up to ‘official’](/#data-security-classification); your app cannot store data classified as 'secret' or 'top secret'
+- be built using a supported [buildpack](/#buildpacks) or written in one of these languages:
+    - Go
+    - Nodejs
+    - Java
+    - PHP
+    - Python
+    - Ruby
+    - static HTML/CSS/Javascript site
+
+[Cloud Foundry buildpacks](/#buildpacks) provide runtime and framework support for your application. For most languages, you will need to provide configuration files to describe your app’s dependencies. Most buildpacks will support a limited range of versions of the language.
+
+# Get started
+
+## Get an account
+
+GOV.UK PaaS is currently in private beta.
```

Comment:
Is there a reason to repeat this information again?

Reply:
nope